test/api/unit/services/SQS.test.js

```diff
@@ -80,8 +80,6 @@ describe('SQS', () => {
     context("building a site's default branch", () => {
       it('should set an empty string for BASEURL in the message for a site with a custom domain', (done) => {
         const domains = [
-          'www.example.com',
```

Comment:
Yeah, there should be a test for the SQS service that verifies that it sets the baseurl correctly for protocol-less domains and protocol-less domains without a trailing slash.